openfisca_core/formula_toolbox.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+
+from datetime import date  # noqa analysis:ignore
+
+from numpy import maximum as max_, minimum as min_, logical_not as not_, where, select   # noqa analysis:ignore
+
+from .columns import (  # noqa analysis:ignore
+    AgeCol,
+    BoolCol,
+    DateCol,
+    EnumCol,
+    FixedStrCol,
+    FloatCol,
+    IntCol,
+    PeriodSizeIndependentIntCol,
+    StrCol,
+    )
+from .enumerations import Enum  # noqa analysis:ignore
+from .formulas import (  # noqa analysis:ignore
+    ADD,
+    calculate_output_add,
+    calculate_output_divide,
+    dated_function,
+    DIVIDE,
+    set_input_dispatch_by_period,
+    set_input_divide_by_period,
+    missing_value
+    )
+from .base_functions import (   # noqa analysis:ignore
+    requested_period_added_value,
+    requested_period_default_value,
+    requested_period_last_or_next_value,
+    requested_period_last_value,
+    )
+from .variables import DatedVariable, Variable  # noqa analysis:ignore
+from .formula_helpers import apply_thresholds, switch  # noqa analysis:ignore
+from .periods import MONTH, YEAR, ETERNITY  # noqa analysis:ignore
```

Comment:
The better way would be to stick to the good practices, which are to import only what is needed by the code of the module.

As now, some Python modules are used as "import forwarders", letting other modules to import them in order to benefit from the imports they made... wow!

Adhering to these good practices will make this behavior forbidden.

But this is not in the scope of this PR.